### PR TITLE
update-faq-official-trackers

### PR DESCRIPTION
**Pull request by ShineBrightMeow**
Provide Bingus with answers to two common issues in the support-forum regarding official tracker sets.

### CODE REVIEW

File: BingusApi/faq_config.json [hunk 1]
```diff
@@ -665,6 +665,26 @@
...
+      "title": "Official Trackers - Blue Light of Death (BLOD)",
+      "answer": "Symptoms of this hardware issue that may occur on some of the **official** trackers include solid blue light (not blinking), suddenly stopped working, or suddenly stopped connecting to WiFi/SlimeVR server. Direct message Eiren with your order number and shipping details for instructions of getting your replacement tracker(s).",
+      "matched_questions": [
+        "solid blue light",
+        "tracker stopped working",
+        "tracker stopped connecting",
+        "blod"
+      ]
+    },
+	{
```
Comment (ButterscotchV) [inline BingusApi/faq_config.json]: Inconsistent spaces and tabs (it should be 2 spaces indentation in this file), please fix the indentation on these opening curly brackets.
```suggestion
    {
      "title": "Official Trackers - Blue Light of Death (BLOD)",
      "answer": "Symptoms of this hardware issue that may occur on some of the **official** trackers include solid blue light (not blinking), suddenly stopped working, or suddenly stopped connecting to WiFi/SlimeVR server. Direct message Eiren with your order number and shipping details for instructions of getting your replacement tracker(s).",
      "matched_questions": [
        "solid blue light",
        "tracker stopped working",
        "tracker stopped connecting",
        "blod"
      ]
    },
    {
```

**Review by ButterscotchV — CHANGES_REQUESTED** [review]
Looks good other than indentation, though maybe the shipping details can be specified further? ex. "order number, address, name, and phone number".

File: BingusApi/faq_config.json [hunk 2]
```diff
@@ -665,6 +665,26 @@
...
+      "title": "Official Trackers - Broken/Missing Extension Cables",
+      "answer": "There are two ways of getting replacement cables for your **official** tracker sets: [SlimeVR Store](https://shop.slimevr.dev/) (You will be helping SlimeVR pay for shipping cost) or by direct messaging Eiren with your order number and shipping details.",
+      "matched_questions": [
+        "extension cable issue",
+        "extension cable broke",
+        "extension cable snap",
+        "extension cable missing"
```
Comment (ButterscotchV) [inline BingusApi/faq_config.json]: Immediate afterthought as well, these will probably resolve to approximately the same vectors, maybe some more variety can be added to this? ex. "my tracker cable is missing from the box", "the extension wire broke when I was using it", "how do I get a replacement cable?"

**Comment by ShineBrightMeow** [comment]
* Fixed Indents to JSON. (NotePad++ made some weird spacing)
* Updated search vectors.
* Updated answers.

**Pull request merged**